tasks/src/main/scala/dagr/tasks/fgbio/CallMolecularConsensusReads.scala

```diff
@@ -36,28 +36,26 @@ import scala.collection.mutable.ListBuffer
  */
 class CallMolecularConsensusReads(val in: PathToBam,
                                   val out: PathToBam,
+                                  val minReads: Int,
                                   val rejects: Option[PathToBam] = None,
                                   val tag: Option[String] = None,
                                   val readNamePrefix: Option[String] = None,
                                   val readGroupId: Option[String] = None,
                                   val errorRatePreUmi: Option[Int] = None,
                                   val errorRatePostUmi: Option[Int] = None,
-                                  val minInputBaseQuality: Option[Int] = None,
-                                  val minConsensusBaseQuality: Option[Int] = None,
```

Comment:
Do you want to deprecate it or remove it completely, you've made a choice, so I thought I would at least mention it.